ts/store/reducers/entities/messages/messagesStatus.ts

```diff
@@ -163,4 +163,19 @@ export const isMessageRead = (
     .map(ms => ms.isRead)
     .getOrElse(false);
 
+/**
+ * Retrieve the MessageStatus given an ID.
+ * Fallback to all false if not found.
+ *
+ * @param state
+ * @param messageId
```

Comment:
```suggestion
```
`messageId` isn't a params of `getMessageStatus` but it is a param of the returned function